web/packages/core/src/ruffle-player.ts

```diff
@@ -809,6 +809,49 @@ export class RufflePlayer extends HTMLElement {
             console.error("SWF download failed");
         }
     }
+    private openVirtualKeyboard(): void {
+        const initialValue = "Initial";
+        const initialValueLength = initialValue.length;
+        const existingInput = this.shadow.getElementById(
+            "ruffle-virtual-keyboard"
+        );
+        if (!existingInput) {
+            const input = document.createElement("input");
+            input.setAttribute("type", "text");
+            input.setAttribute("autocapitalize", "off");
+            input.style.position = "absolute";
+            input.style.top = "-100px";
+            input.style.width = "1px";
+            input.style.height = "1px";
+            input.id = "ruffle-virtual-keyboard";
```

Comment:
Probably should make a variable to hold this magic string `"ruffle-virtual-keyboard"`.